### ISSUE TITLE
Chart cannot be repainted after switching perspective

### ISSUE THREAD

**Issue by benoitgaudou** [gama-platform/gama#2639]
**Describe the bug**
Run a model with charts, launch the simulation.
Switch the perspective from simulation to modeling and switch again from modeling and simulation.
The display is white, and not repaint.

See an example:
```
model testCharts


experiment exp type: gui {
//	layout #split ;
	output
	{
 		display "style_cumulative_style_chart" type: java2D
		{
			chart "Style Cumulative chart" type: series
			{
				data "Spline" value: cycle color: # orange style: spline;
			}
		} 
	}
}
```

**Desktop (please complete the following information):**
 - OS: macOS Mojave
 - GAMA version: git

**Comment by AlexisDrogoul** [comment]
Seems to be happening on macOS with the Java2D displays in general...

**Comment by pcaillou** [comment]
Ohhh a nice little bug so I can feel I am in the coding camp :)
Thanks, looking into it!

Philippe

**Comment by pcaillou** [comment]
I can't reproduce it with windows, mac os specific?

**Comment by AlexisDrogoul** [comment]
macOS specific, yes...

**Comment by benoitgaudou** [comment]
It seems to have been fixed by 42d958afec776f3eb8d48530847eaff6a3f153ea
